tests/test_profile_general.py

```diff
@@ -319,7 +319,7 @@ def gpu_soc():
 
 # Set rocprofv2 as profiler if MI300
 if "MI300" in soc:
```

Comment:
For the azure job running on gfx942 specifically, I think soc would return "MI308" here.

Reply:
MI300X_A1. I confirmed via chip id